example/per.jonprl

```diff
@@ -0,0 +1,125 @@
+Operator per-squash : (0).
+Prefix 7 "⇃" := per-squash.
+[⇃ A] =def= [per(lam(_.lam(_.A)))].
```

Comment:
Ah, OK! Sounds good :)

This is, by the way, a really nice version of the squash type. Is there any advantage of the `image`-based one over this (since it looks like they are equivalent)?